Demo/RCHAccountViewController.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  RCHAccountViewController.swift
+//  RichRelevanceSDKDemo
+//
+//  Created by Ariana Antonio on 10/6/16.
+//  Copyright © 2016 Raizlabs Inc. All rights reserved.
+//
+
+import UIKit
+
+class RCHAccountViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+
+    @IBOutlet var accountTableView: UITableView!
+    @IBOutlet var userIDTextField: UITextField!
+    @IBOutlet var modalView: UIView!
+    @IBOutlet var saveUserButton: UIButton!
+    @IBOutlet var userCellLabel: UILabel!
+    
+    var userIDArray: [String]?
```

Comment:
#pp 
One approach I like is to model the state the view controller uses and have `didSet` manage the binding to the view layer.  This is scenario a bit different since the storage is in `UserDefaults`, and would use `set`, but for instance:

```swift
var userIDs: [String] {
    get {
        return UserDefaults.standard.object(forKey: kRCHUserDefaultKeyUserIDs) as? [String] ?? []
    }
    set {
        UserDefaults.standard.set(newValue, forKey: kRCHUserDefaultKeyUserIDs)
        UserDefaults.standard.synchronize
        accountTableView.reloadData()
    }
}
var currentUser: String {
    // Same sort of relay into UserDefaults and reloadData()
}
```
Then else where you can things like:

```swift
userIDs.append(newUser)
currentUser = newUser
```

And the view is updated as a side effect of managing state on the view controller. It will trigger `reloadData` twice, which isn't ideal, but for these scenarios should be pretty inconsequential, and there are some ways around that. Another application of this approach would be to have a `var displayModal: Bool` and in `didSet` update the hidden state and manage the first responder.